packages/caliper-fabric/lib/fabric.js

```diff
@@ -15,13 +15,12 @@
 'use strict';
 
 const FabricClient = require('fabric-client');
+let FabricNetworkAPI = require('fabric-network');
```

Comment:
We shouldn't forget to add `fabric-network` as a dependency for all SUT bindings, otherwise, this will fail runtime.